src/main/groovy/info/solidsoft/gradle/pitest/PitestTask.groovy

```diff
@@ -18,12 +18,7 @@ package info.solidsoft.gradle.pitest
 import groovy.transform.CompileStatic
 import groovy.transform.PackageScope
 import org.gradle.api.file.FileCollection
-import org.gradle.api.tasks.Input
-import org.gradle.api.tasks.InputFiles
-import org.gradle.api.tasks.JavaExec
-import org.gradle.api.tasks.Optional
-import org.gradle.api.tasks.OutputDirectory
-import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.*
```

Comment:
I don't like *-imports in general. It can hide the real number of imported classes in a given class.